packages/common/babel.config.js

```diff
@@ -183,6 +183,52 @@ const carbonMapper = (importName) => ({
   ProgressStep: 'ProgressIndicator'
 }[importName] || importName)
 
+const createCarbonCJSTransformList = (env) => [
+  'transform-imports',
+  {
+    'carbon-components-react/es/components/StructuredList/StructuredList': {
+      transform: (importName) => {
+        let res;
+        const files = glob.sync(path.resolve(__dirname, `../../node_modules/carbon-components-react/${env === 'cjs' ? 'lib' : 'es' }/**/${carbonMapper(importName)}.js`));
+        if (files.length > 0) {
+          res = files[0];
+        } else {
+          throw new Error(`File with importName ${importName} does not exist`);
+        }
+        res = res.replace(path.resolve(__dirname, '../../node_modules/'), '');
+        res = res.replace(/^\//, '');
+        return res;
+      },
+      preventFullImport: false,
+      skipDefaultConversion: true
+    }
+  },
+  `carbon-components-react-structured-list-${env}`
+];
+
+const createCarbonCJSTransformProgress = (env) => [
+  'transform-imports',
+  {
+    'carbon-components-react/es/components/ProgressIndicator/ProgressIndicator': {
```

Comment:
you can add these entries to the existing transform mapper. No need to create an extra special new plugin.

Reply:
you are right